pkg/cvo/availableupdates.go

```diff
@@ -187,40 +192,71 @@ func calculateAvailableUpdatesStatus(clusterID string, proxyURL *url.URL, tlsCon
 	currentVersion, err := semver.Parse(version)
 	if err != nil {
 		klog.V(2).Infof("Unable to parse current semantic version %q: %v", version, err)
-		return nil, configv1.ClusterOperatorStatusCondition{
+		return cvoCurrent, nil, configv1.ClusterOperatorStatusCondition{
 			Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: "InvalidCurrentVersion",
 			Message: "The current cluster version is not a valid semantic version and cannot be used to calculate upgrades.",
 		}
 	}
 
-	updates, err := cincinnati.NewClient(uuid, proxyURL, tlsConfig).GetUpdates(upstreamURI, arch, channel, currentVersion)
+	current, updates, err := cincinnati.NewClient(uuid, proxyURL, tlsConfig).GetUpdates(upstreamURI, arch, channel, currentVersion)
 	if err != nil {
 		klog.V(2).Infof("Upstream server %s could not return available updates: %v", upstream, err)
 		if updateError, ok := err.(*cincinnati.Error); ok {
-			return nil, configv1.ClusterOperatorStatusCondition{
+			return cvoCurrent, nil, configv1.ClusterOperatorStatusCondition{
 				Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: updateError.Reason,
 				Message: fmt.Sprintf("Unable to retrieve available updates: %s", updateError.Message),
 			}
 		}
 		// this should never happen
-		return nil, configv1.ClusterOperatorStatusCondition{
+		return cvoCurrent, nil, configv1.ClusterOperatorStatusCondition{
 			Type: configv1.RetrievedUpdates, Status: configv1.ConditionFalse, Reason: "Unknown",
 			Message: fmt.Sprintf("Unable to retrieve available updates: %s", err),
 		}
 	}
 
-	var cvoUpdates []configv1.Update
+	cvoCurrent, err = convertCincinnatiUpdateToRelease(current)
```

Comment:
```suggestion
	cvoCurrent, err = convertRetreivedUpdateToRelease(current)
```

Reply:
This is internal, s)not user-facing, so I think it should stay per [here][1].

[1]: https://github.com/openshift/cluster-version-operator/pull/419#discussion_r462978410

Reply:
`convertRetreivedUpdateToRelease` is a better name than convertCincinnatiUpdateToRelease IMO.

Reply:
2 votes for renaming vs. my one, so renamed with 0a0483ae -> cd2d7013.